internal/transform/scope-css.go

```diff
@@ -85,6 +85,17 @@ outer:
 
 					// if inside @keyframes, don’t transform what’s there
 					if isKeyframes {
+						if strVal == "{" {
+							keyframeCurlyCount++
+						} else if strVal == "}" {
+							keyframeCurlyCount--
+						}
+
+						// Inside of this case, we only want to break out when keyframeCurlyCount is -1
+						// since 0 is the default
+						if keyframeCurlyCount < 0 {
+							isKeyframes = false
+						}
```

Comment:
`keyframeCurlyCount == 0` is the default case, we are done when `keyframeCurlyCount == -1`.